eng/common/TestResources/TestResources-Helpers.ps1

```diff
@@ -1,267 +1,282 @@
 function Log($Message) {
-    Write-Host ('{0} - {1}' -f [DateTime]::Now.ToLongTimeString(), $Message)
+  Write-Host ('{0} - {1}' -f [DateTime]::Now.ToLongTimeString(), $Message)
 }
 
 # vso commands are specially formatted log lines that are parsed by Azure Pipelines
 # to perform additional actions, most commonly marking values as secrets.
 # https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands
 function LogVsoCommand([string]$message) {
-    if (!$CI -or $SuppressVsoCommands) {
-        return
-    }
-    Write-Host $message
+  if (!$CI -or $SuppressVsoCommands) {
+    return
+  }
+  Write-Host $message
 }
 
 function Retry([scriptblock] $Action, [int] $Attempts = 5) {
-    $attempt = 0
```

Comment:
Are the indentation changes relevant? Or can they be removed to reduce the amount of churn in the PR?

Reply:
Dang. I was trying to be careful not to but have to use a different key chord. Apparently, I forgot once.

Alternatively, @weshaggard can we put an `.editorconfig` in the `eng/common` folder to keep these things consistent? Fighting the formatting is a PITA.